39.md

```diff
@@ -62,3 +70,79 @@ Identity: A Telegram user ID.
 
 Proof: A string in the format `<ref>/<id>` which points to a message published in the public channel or group with name `<ref>` and message ID `<id>`. This message should be sent by user ID `<identity>` and have the text `Verifying that I control the following Nostr public key: "<npub encoded public key>"`.
 This can be located at `https://t.me/<proof>`.
+
+### `openpgp4fpr`
+
+Identity: A PGP fingerprint as described in the [openpgp4fpr URI scheme](https://metacode.biz/openpgp/openpgp4fpr), uppercase.
```

Comment:
This should be case insensitive as described in the linked document.

Reply:
Also, ideally, mentions of `PGP` should instead be `OpenPGP`, though I'll admit I'm being pedantic here.

Reply:
Are relay queries case-insensitive? Probably not. That's all I care about.

Reply:
Interesting point, I don't know the answer.

Reply:
There are a lot of mentions of "lowercase" in this repo, and also this NIP contained:
> Identity names SHOULD be normalized if possible by replacing uppercase letters with lowercase letters, and if there are multiple aliases for an entity the primary one should be used.

I'll update to reflect lowercase. It costs nothing to lowercase everything before publishing and makes it easy to query.